styles/styles.css

```diff
@@ -125,6 +134,14 @@
   --line-width-thin: 2px;
   --line-width-extra-light: 3px;
 
+  /* Line Height */
```

Comment:
should we pass line heights as relative to the underlying font size for example 
`line-height: 1.6` 
if there is hardcoded height we can consider adding/using --space-XXpx pattern

Reply:
I guess line height we can keep the english nomecluture instead of px, looks more intutive